aurelia_project/environments/dev.ts

```diff
@@ -96,6 +96,7 @@ export default {
       showInspectPanel: 'inspectCorrelation:inspectPanel:show',
       showTokenViewer: 'inspectCorrelation:tokenViewer:show',
       showLogViewer: 'inpsectCorrelation:logViewer:show',
+      noCorrelationsFound: 'inspectCorrelation:diagramViewer:noCorrelationsFound'
```

Comment:
Missing trailing comma

```suggestion
      noCorrelationsFound: 'inspectCorrelation:diagramViewer:noCorrelationsFound',
```